src/binding/abi/mpi_abi.h

```diff
@@ -1881,6 +1904,29 @@ MPI_Session PMPI_Session_f2c(MPI_Fint session);
 MPI_Fint PMPI_Win_c2f(MPI_Win win);
 MPI_Win PMPI_Win_f2c(MPI_Fint win);
 
+int PMPI_Comm_toint(MPI_Comm comm);
+MPI_Comm PMPI_Comm_fromint(int comm);
+int PMPI_Errhandler_toint(MPI_Errhandler errhandler);
+MPI_Errhandler PMPI_Errhandler_fromint(int errhandler);
+int PMPI_Type_toint(MPI_Datatype datatype);
```

Comment:
Minor nit: in https://github.com/mpi-forum/header_and_stub_library/pull/39, I'm list the `Type` routines after `Session` and before `Win` to keep the listing sorted alphabetically.

Reply:
Thanks for checking! I'll sync it from upstream later.